src/network/networkd-address.c

```diff
@@ -32,6 +32,8 @@
 #include "utf8.h"
 #include "util.h"
 
+#define STATIC_ADDRESSES_PER_NETWORK_MAX 1024U
```

Comment:
Hm, what's the point of this limit? Isn't adding addresses/routes/fdb entries already done by trusted code anyway? An artificial limit like this can hit in more complicated setups, and the accounting makes the code more complicated too.